creme/creme_core/tests/utils/test_main.py

```diff
@@ -104,7 +104,7 @@ def test_truncate_str_01(self):
 
     def test_create_if_needed(self):
         title = 'Mister'
-        pk = 1024
+        pk = 999999
```

Comment:
Missing refactoring to UNUSED_PK ??

Reply:
It's a create_if_needed so we need an eligible (< 2147483647 int) PK here

Reply:
Ok ; so a comment would be nice to avoid a silly change in the future